CameraXBasic/app/src/main/java/com/android/example/cameraxbasic/fragments/CameraFragment.kt

```diff
@@ -382,17 +388,23 @@ class CameraFragment : Fragment() {
             imageCapture?.let { imageCapture ->
 
                 // Create output file to hold the image
-                val photoFile = createFile(outputDirectory, FILENAME, PHOTO_EXTENSION)
+                val photoFile: File = createFile(outputDirectory, FILENAME, PHOTO_EXTENSION)
 
                 // Setup image capture metadata
                 val metadata = Metadata().apply {
-
                     // Mirror image when using the front camera
                     isReversedHorizontal = lensFacing == CameraSelector.LENS_FACING_FRONT
                 }
 
-                // Setup image capture listener which is triggered after photo has been taken
-                imageCapture.takePicture(photoFile, metadata, mainExecutor, imageSavedListener)
+                // Setup ImageCapture output options
+                // For saving to ContentProvider, it's Builder(ContentResolver, Uri, ContentValues)
```

Comment:
I like this comment, I think it gives developers a hint without overwhelming them with additional files or flags